<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionproduit</class>
 <widget class="QDialog" name="gestionproduit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="gestion_PRODUIT">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>731</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/img/background-smh-avion.jpg);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>gestion produit</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>671</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Afficher_produit</string>
      </attribute>
      <widget class="QTableView" name="tab_produit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>681</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Ajouter_produit</string>
      </attribute>
      <widget class="QPushButton" name="Ajouter_produit">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>350</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>AJOUTER</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>431</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>DF</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_typeproduit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_DEXproduit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>DEX</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>id:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>id_fournisseur:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_DFproduit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_idproduit">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Modifier_produit</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>421</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_modtypeproduit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_modDFproduit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>id_fournisseur:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>DF</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_modDEXproduit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_modidproduit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>DEX</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>id:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxmodidproduitfournisseur"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="modifierproduit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>380</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Supprimer_produit</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_suppproduit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_produit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>gestion fournisseur</string>
    </attribute>
    <widget class="QTabWidget" name="gestion_fournisseur">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>471</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QPushButton" name="ajouter_fournisseur">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>411</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_idfournisseur"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>nom</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_nomfournisseur"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>mail</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_mail"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>afficher</string>
      </attribute>
      <widget class="QTableView" name="tab_fournisseur">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>621</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_suppfournisseur">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>id :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_fournisseur">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>411</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>nom</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_modnomfournisseur"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>mail</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_modmailfournisseur"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_modidfournisseur"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="modifierfournisseur">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>rechercher</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>361</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>RECHERCHER</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>300</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>mailing</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>361</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::WrapLongRows</enum>
        </property>
        <property name="verticalSpacing">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Smtp-server:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="server">
          <property name="text">
           <string>smtp.gmail.com</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Server port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="port">
          <property name="text">
           <string>465</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="uname"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="paswd">
          <property name="inputMask">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Recipant to:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="rcpt"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Subject:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="subject"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Message:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPlainTextEdit" name="msg"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="sendBtn">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>send</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exitBtn">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>350</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>statistique</string>
      </attribute>
      <widget class="QTableView" name="tablestat">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>21</y>
         <width>471</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="consulter_stat">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>consulter</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
